src/Symfony/Component/Translation/MessageCatalogue.php

```diff
@@ -19,7 +19,7 @@
  */
 class MessageCatalogue implements MessageCatalogueInterface, MetadataAwareInterface
 {
-    private $messages = [];
+    private $messages;
```

Comment:
Could you elaborate why you made this change?

Reply:
I think that it's a PHPStorm auto fix because it a redundant initialization of the property.

Reply:
Let's revert this to reduce the number of changes in this PR.